migrations/sqls/20190910195603-services-update-09102019-up.sql

```diff
@@ -0,0 +1,37 @@
+INSERT INTO programs (programName, programDescription)
+VALUES
+	("Focus Groups", "Focus Groups")
+;
+
+DELETE FROM services
+WHERE serviceName = "Youth Groups / Leadership Development" AND programeName = "Preventive Health"
```

Comment:
typo: `programName`

This makes me nervous that this migration hasn't been tested -- as it is right now it would throw an error if executed. Have you been able to test that these work? With `npx db-migrate up`

Reply:
No, sorry I meant to send push this as a draft, not a ready PR. This was the one I was having trouble getting the migrations to work.